Pocket/pocket-ios
# fix(refreshing): updating refresh coordinators to check for a session

## Summary

This updates the refresh coordinators to not load data if there is not an active session on the device.

## Implementation Details
* Injected the session into the coordinators for refreshing

## Test Steps
* Ensure that when you are logged out and open the app, you see logs that the saves and home are not loading.


## PR Checklist:

## Screenshots
